gtsam/gtsam.i

```diff
@@ -2759,21 +2759,35 @@ virtual class EssentialMatrixFactor : gtsam::NoiseModelFactor {
 };
 
 #include <gtsam/slam/dataset.h>
+// Dummy classes, for MATLAB wrappers
+class SfmMeasurement{};
+class SiftIndex{ };
+class SfmMeasurements{};
+class SfmCamera{};
```

Comment:
I think the MATLAB wrapper also needs them to have basic methods to do anything with them. Like at() or push_back() for vectors. That is the case for other vectors that have been wrapped in this file.

Reply:
i haven't implemented the matlab wrapper yet. These are dummy classes